File: go/utility/feature_add.go
package utility

import "github.com/voxgig-sdk/passantenfrequenz-stadt-stgallen-sdk/core"

func featureAddUtil(ctx *core.Context, f core.Feature) {
	client := ctx.Client
	features := client.Features

	client.Features = append(features, f)
}

File: php/utility/FeatureAdd.php
<?php
declare(strict_types=1);

// PassantenfrequenzStadtStgallen SDK utility: feature_add

class PassantenfrequenzStadtStgallenFeatureAdd
{
    public static function call(PassantenfrequenzStadtStgallenContext $ctx, mixed $f): void
    {
        $ctx->client->features[] = $f;
    }
}
